### PR TITLE
JavaScript Capstone Project

### PR DESCRIPTION
**Pull request by egichuhis**
This project was done by @egichuhis  and @lucy-sees .

Please review this PR. 

Changes made:

- **List structure layout**
- **Fetch TV Shows from API**
- **Display the fetched TV Shows**
- **Like a TV Show(s)**
- **Increment like counter and update view**
- **Popup window to show more TV Show details**
- **Post comments to Involvement API**
- **Fetch comments from API and display**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,198 @@
...
+
+- GitHub: [@githubhandle](https://github.com/egichuhis)
+- Twitter: [@twitterhandle](https://twitter.com/egichuhis)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/edwin-gichuhi/)
+
+👤 **Lucy W. Mwangi**
+
+- GitHub: [@lucy-sees](https://github.com/lucy-sees)
+- Twitter: [@\_The_LoneWolf](https://twitter.com/_The_LoneWolf)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/lucy-w-mwangi-83a31b11b/)
```
Comment (Efiamotu-1) [inline README.md]: - [x] This link is currently broken as when I click on it, it displays an error 404 page, please kindly provide the appropriate link as it is unprofessional to leave broken links in your readme file 👍 ✨

File: src/modules/getShowsCount.js
Comment (Efiamotu-1) [inline src/modules/getShowsCount.js:1]: - [x] The project requirement shown belows says that your home page is expected to display the count of movies which you have on the webpage, seeing that you currently have 15 movies on your homepage, this number should be displayed somewhere to meet this requirement 👍.
mt
- [x] For the comments section even when I add a comment the comment count is not updated as shown in the image below, please kindly update this 👍 
cm

**Review by Efiamotu-1 — CHANGES_REQUESTED** [review]
Hi @egichuhis  & @lucy-sees , 

 

Good job so far! 👏. Your Code is Awesome! 💪

There are some **few** issues 🐛 that you still need to work 🔧 on though to go to the next project ⏭️ but you are almost there! 🏃
## Don't Worry 🙏

It's ok to have bugs in your code 🐛. Please, see this code review as a **chance** for you to **learn** practical coding skills and learn **new tips** that can improve your code quality 💯 and help you **get a better ** 💼.

_I am going to guide you on how to fix them down below. 🤓_

## Highlights :dart: 
- ✔️ No linter Errors
- ✔️ Correct gitflow
- ✔️ Good job adding tv maze & involvement api
- ✔️ Good job adding tests 
- ✔️ Descriptive README
- ✔️ Descriptive PR title and description
- The code is top-notch! 💯 and has a huge potential.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


## Important Notes 📝

- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

> **Note:** Please, don't Merge This Pull Request yet. Only Merge when it is Approved ✅.


## Thank you for your time 🙏 and call me if you need help or clarification ☎️

- This is My **GitHub**: [@Efiamotu-1](https://github.com/Efiamotu-1/)
- This is My **Twitter**: [@EFYAMOTU](https://twitter.com/EFYAMOTU)
- This is My **linkedIn**.[Habeeb Efiamotu Musa](https://www.linkedin.com/in/musa-habeeb/)
- This is My **Slack**: [Habeeb Efiamotu Musa](https://slack.com)

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by brainconnect93 — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi @egichuhis & @lucy-sees, 👋 👋  

Your project is complete! Great job for following the project requirement🥇and implementing the changes requested by the previous reviewer. There is nothing else to say other than... it's time to merge it 🍾🚢 :shipit: . Congratulations!  🎉💯🌟 




## Good Points 👍

- Descriptive Pull Request title & summary. ✔️
- No Linter errors ✔️
- Comment detail popup :heavy_check_mark: 
- like counter works perfectly :heavy_check_mark: 
- Involvement API used correctly :heavy_check_mark: 
- Pair-programming capstone project :heavy_check_mark: 
- Good use of Git flow ✔️

## Optional changes

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers..Happy coding!..and keep soaring higher! 💻 🍷 🚀 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @brainconnect93 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**